hack/generate-lib-resources.py

```
@@ -150,19 +151,88 @@ def generate_resourcebuilder(directory, types, clients, modifiers, health_checks
+        '}',
+        '',
+        'func newBuilder(config *rest.Config, m manifest.Manifest) (Interface, error) {',
+        '\tcs, err := getOrCreateClientSet(config)',
+        '\tif err != nil {',
+        '\t\treturn nil, err',
+        '\t}',
+        '\treturn &builder{',
+        '\t\traw: m.Raw,',
+        '',
+    ])
+    for prop_name, data in sorted(client_properties.items()):
```

Ruff (0.16.0)

[warning] Loop control variable `data` not used within loop body

(B007)